src/arrayMethodSort.js

```diff
@@ -4,8 +4,27 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (
+    compareFunction = (a, b) => {
+      if (a + '' < b + '') {
+        return -1;
+      }
+
+      return 1;
+    },
+  ) {
+    for (let i = 0; i < this.length; i++) {
+      for (let j = 0; j < this.length; j++) {
```

Comment:
The inner loop should start from `i + 1` instead of `0` to avoid unnecessary comparisons and potential swaps with already sorted elements.

Reply:
I find it more convenient to use i !== j than j = i + 1 and then deal with the condition i < this.length and j < this.length to make everything work correctly. If it is wrong, I would like to know why